In [4]:
import pandas as pd
import matplotlib
pd.set_option('display.max_rows', 500)
pd.set_option('display.max_columns', 500)
pd.set_option('display.width', 1000)

In [6]:
df_auslaender = pd.read_csv('auslaender_bern.csv')

In [7]:
df_auslaender.head(10)

quartier_stadtteil  total_2018  deutschland  mazedonien  spanien  frankreich  italien  eritrea  kosovo  kroatien  oesterreich  polen  portugal  serbien  tuerkei  uk  somalia  usa  china  indien  srilanka  syrien
0  Schwarzes Quartier         166           68           0        6           5       23        0       0         0            4      1         1        1        0   3        0    5      0       0         5       0
1    Weisses Quartier         141           40           0        7           8       15        0       0         1            4      1         3        2        2   5        0    6      0       0         0       0
2     Grünes Quartier         313           94           0        9          27       34        0       0         2            9      6         8        2        5  14        0   14      3       0         1       3
3     Gelbes Quartier         150           27           0        2           6       31        2       2         1            5      2         5        1        7   4        2    3      0       1         7       0
4      Rotes Quartier         120           24           0        5           2       14        0       0         0            0      6         3        0        3   1        0   10      2       0         0       1
5        Innere Stadt         890          253           0       29          48      117        2       2         4           22     16        20        6       17  27        2   38      5       1        13       4
6            Engeried         192           54           1       23          19       16        0       4         3            8      2         7        0        2   1        2    3      0       0         0       0
7            Felsenau         714          152          30       20          16       55       24      19        17           11      8        18       29       27   8        2    4      7      16        39       9
8             Neufeld         894          270           0       54          43      119       42       2         4           41      7        17       11        7  15        2    8     15       6        18       4
9           Länggasse         596          171           1       39          25       97       11       9         9           19      7        17        8        5   8        3    9     16       6         2       2

In [15]:
df_auslaender.tail(10)

quartier_stadtteil  total_2018  deutschland  mazedonien  spanien  frankreich  italien  eritrea  kosovo  kroatien  oesterreich  polen  portugal  serbien  tuerkei   uk  somalia  usa  china  indien  srilanka  syrien
29             Breitfeld        1281          206          18      116          36      186       22      44        33           21     27        63       50       36    9        1   12     15       9        30       4
30           Breitenrain        1384          235          15      101          53      179       45      91        22           28     17        54       16       43   13        5   10     19       8        25       5
31              Lorraine         789          169          24       58          14      114       20      34         6           13      8        23        9       19    6       23    5      7       4        41      20
32  Breitenrain-Lorraine        4837         1038          58      373         150      614      110     182        72          114     82       173       85      120   56       30   62     74      30       126      47
33               Bümpliz        5612          445         443      358          56      877      203     350       120           68     55       370      155      269   37      111   24     97      50       182      78
34          Oberbottigen         103           31           1        9           5       20        0       0         1            2      1         4        0        2    3        0    0      0       0         5       0
35            Stöckacker         717           99          16       56           3       78       28      34        14            7      3        43       25       46    3       32    3     15       4        35       1
36             Bethlehem        5757          359         558      319          34      758      167     418        89           51     56       459      202      388   18      166   15     94      56       112     132
37  Bümpliz-Oberbottigen       12189          934        1018      742          98     1733      398     802       224          128    115       876      382      705   61      309   42    206     110       334     211
38       stadtbern_total       34375         6345        1238     1970         863     4200      841    1230       471          649    463      1551      737     1182  420      474  371    653     391       812     474

In [8]:
df_auslaender.shape

(39, 22)

In [13]:
df_auslaender.dtypes

quartier_stadtteil    object
total_2018             int64
deutschland            int64
mazedonien             int64
spanien                int64
frankreich             int64
italien                int64
eritrea                int64
kosovo                 int64
kroatien               int64
oesterreich            int64
polen                  int64
portugal               int64
serbien                int64
tuerkei                int64
uk                     int64
somalia                int64
usa                    int64
china                  int64
indien                 int64
srilanka               int64
syrien                 int64
dtype: object

## Analyse Ausländer in Bern 

### Top 20 Nationalitäten in Bern 

In [51]:
df_auslaender[df_auslaender.quartier_stadtteil == 'stadtbern_total']

quartier_stadtteil  total_2018  deutschland  mazedonien  spanien  frankreich  italien  eritrea  kosovo  kroatien  oesterreich  polen  portugal  serbien  tuerkei   uk  somalia  usa  china  indien  srilanka  syrien
38    stadtbern_total       34375         6345        1238     1970         863     4200      841    1230       471          649    463      1551      737     1182  420      474  371    653     391       812     474

Für Tabelle Top 20:
1. Deutschland: 6345
2. Italien: 4200
3. Spanien: 1970
4. Portugal: 1551
5. Mazedonien: 1238
6. Kosovo: 1230
7. Türkei: 1182
8. Frankreich: 863
9. Eritrea: 841
10. Sri Lanka: 821
11. Serbien: 737
12. China: 653
13. Österreich: 649
14. Somalia: 474
15. Syrien: 474
16. Kroatien: 471
17. Polen: 463
18. Vereinigtes Königreich: 420
19. Indien: 391
20. Vereinigte Staaten: 371

### 1. Deutsche 

In [23]:
#Beliebteste Quartiere für Deutsche:
#1. Mattenhof-Weissenbühl 1800
#2. Kirchenfeld-Schosshalde 1288
#3. Breitenrain-Lorraine 1038
#4. Länggasse-Felsenau 1032
df_auslaender.sort_values(by='deutschland', ascending=False)

quartier_stadtteil  total_2018  deutschland  mazedonien  spanien  frankreich  italien  eritrea  kosovo  kroatien  oesterreich  polen  portugal  serbien  tuerkei   uk  somalia  usa  china  indien  srilanka  syrien
38          stadtbern_total       34375         6345        1238     1970         863     4200      841    1230       471          649    463      1551      737     1182  420      474  371    653     391       812     474
19    Mattenhof-Weissenbühl        7189         1800          73      374         252      812      162     119        82          153    121       215      117      181   98       77   88    168      99       201     101
26  Kirchenfeld-Schosshalde        5501         1288          44      240         175      483       63      76        42          119     87       169       89      101  124       41  100    149      96        62      56
32     Breitenrain-Lorraine        4837         1038          58      373         150      614      110     182        72          114     82       173       85      120   56       30   62     74      30       126      47
12       Länggasse-Felsenau        3769         1032          45      212         140      441      106      49        47          113     42        98       58       58   54       15   41     51      55        76      55
37     Bümpliz-Oberbottigen       12189          934        1018      742          98     1733      398     802       224          128    115       876      382      705   61      309   42    206     110       334     211
24              Schosshalde        1842          446          24       72          61      191       13      36        16           41     20        57       58       38   49        7   29     44      19        20      24
33                  Bümpliz        5612          445         443      358          56      877      203     350       120           68     55       370      155      269   37      111   24     97      50       182      78
15                Mattenhof        1671          440          21       80          48      235       29      16        17           39     25        50       20       36   18        9   10     37      14        73      16
13                 Holligen        2153          428          24      130          46      248       59      60        45           38     35       107       25       68   22       23   19     70      60        80      49
17              Weissenbühl        1483          404           7       77          46      160       39      20         7           36     26        29       43       41   36        4   14     32      14        20      13
36                Bethlehem        5757          359         558      319          34      758      167     418        89           51     56       459      202      388   18      166   15     94      56       112     132
28              Spitalacker        1155          347           1       80          39      122       20      13         9           36     23        23        9       20   24        1   24     33       9        24      18
11                 Muesmatt         945          278          12       58          25      121       14      13        10           25     16        35        8       13   13        4   11      9      26        12       9
8                   Neufeld         894          270           0       54          43      119       42       2         4           41      7        17       11        7   15        2    8     15       6        18       4
5              Innere Stadt         890          253           0       29          48      117        2       2         4           22     16        20        6       17   27        2   38      5       1        13       4
20              Kirchenfeld         734          250           9       52          19       50        7       2         2           20      9         9        1        5   20        1   24     12       5         0       4
30           

In [24]:
#Konzentration in Stadtteilen/Quartieren in Prozent:
(100/6345)*1800 #In Mattenhof-Weissenbühl

28.368794326241133

In [26]:
(100/6345)*1288 #In Kirchenfeld-Schosshalde 

20.299448384554765

In [27]:
(100/6345)*1038 #In Breitenrain-Lorraine  

16.35933806146572

In [29]:
(100/6345)*1032 #In Länggasse-Felsenau 

16.264775413711583

### 2. Mazedonier 

In [16]:
# Beliebteste Quartiere für Mazedonier:
#1. Bethlehem 558
#2. Bümpliz  443
df_auslaender.sort_values(by='mazedonien', ascending=False).head(10)

quartier_stadtteil  total_2018  deutschland  mazedonien  spanien  frankreich  italien  eritrea  kosovo  kroatien  oesterreich  polen  portugal  serbien  tuerkei   uk  somalia  usa  china  indien  srilanka  syrien
38          stadtbern_total       34375         6345        1238     1970         863     4200      841    1230       471          649    463      1551      737     1182  420      474  371    653     391       812     474
37     Bümpliz-Oberbottigen       12189          934        1018      742          98     1733      398     802       224          128    115       876      382      705   61      309   42    206     110       334     211
36                Bethlehem        5757          359         558      319          34      758      167     418        89           51     56       459      202      388   18      166   15     94      56       112     132
33                  Bümpliz        5612          445         443      358          56      877      203     350       120           68     55       370      155      269   37      111   24     97      50       182      78
19    Mattenhof-Weissenbühl        7189         1800          73      374         252      812      162     119        82          153    121       215      117      181   98       77   88    168      99       201     101
32     Breitenrain-Lorraine        4837         1038          58      373         150      614      110     182        72          114     82       173       85      120   56       30   62     74      30       126      47
12       Länggasse-Felsenau        3769         1032          45      212         140      441      106      49        47          113     42        98       58       58   54       15   41     51      55        76      55
26  Kirchenfeld-Schosshalde        5501         1288          44      240         175      483       63      76        42          119     87       169       89      101  124       41  100    149      96        62      56
7                  Felsenau         714          152          30       20          16       55       24      19        17           11      8        18       29       27    8        2    4      7      16        39       9
13                 Holligen        2153          428          24      130          46      248       59      60        45           38     35       107       25       68   22       23   19     70      60        80      49

In [19]:
# Konzentration von Mazedoniern in Stadtteil Bümpliz-Oberbottigen in Prozent:
(100/1238)*1018 # in Bümpliz-Oberbottigen

82.22940226171244

In [20]:
(100/1238)* 558 # in Bethlehem

45.07269789983845

In [21]:
(100/1238)* 443 # in Bümpliz

35.78352180936995

### 3. Spanier 

In [75]:
# Beliebteste Stadtteile für Spanier:
#1. Bümpliz-Oberbottigen
#2. Mattenhof-Weissenbühl
#3. Breitenrain-Lorraine

# Quartiere
#1. Bümpliz
#2. Bethlehem
#3. Holligen
#4. Breitfeld
#5. Breitenrain

df_auslaender.sort_values(by='spanien', ascending=False).head(10)

quartier_stadtteil  total_2018  deutschland  mazedonien  spanien  frankreich  italien  eritrea  kosovo  kroatien  oesterreich  polen  portugal  serbien  tuerkei   uk  somalia  usa  china  indien  srilanka  syrien
38          stadtbern_total       34375         6345        1238     1970         863     4200      841    1230       471          649    463      1551      737     1182  420      474  371    653     391       812     474
37     Bümpliz-Oberbottigen       12189          934        1018      742          98     1733      398     802       224          128    115       876      382      705   61      309   42    206     110       334     211
19    Mattenhof-Weissenbühl        7189         1800          73      374         252      812      162     119        82          153    121       215      117      181   98       77   88    168      99       201     101
32     Breitenrain-Lorraine        4837         1038          58      373         150      614      110     182        72          114     82       173       85      120   56       30   62     74      30       126      47
33                  Bümpliz        5612          445         443      358          56      877      203     350       120           68     55       370      155      269   37      111   24     97      50       182      78
36                Bethlehem        5757          359         558      319          34      758      167     418        89           51     56       459      202      388   18      166   15     94      56       112     132
26  Kirchenfeld-Schosshalde        5501         1288          44      240         175      483       63      76        42          119     87       169       89      101  124       41  100    149      96        62      56
12       Länggasse-Felsenau        3769         1032          45      212         140      441      106      49        47          113     42        98       58       58   54       15   41     51      55        76      55
13                 Holligen        2153          428          24      130          46      248       59      60        45           38     35       107       25       68   22       23   19     70      60        80      49
29                Breitfeld        1281          206          18      116          36      186       22      44        33           21     27        63       50       36    9        1   12     15       9        30       4

In [76]:
(100/1970)*742 #Bümpliz-Oberbottigen

37.66497461928934

In [77]:
(100/1970)*374 #Mattenhof-Weissenbühl

18.98477157360406

In [78]:
(100/1970)*373 #Breitenrain-Lorraine

18.934010152284262

### 4. Vereinigtes Königreich 

In [80]:
# Beliebteste Stadtteile für UK:
#1. Kirchenfeld-Schosshalde
#2. Mattenhof-Weissenbühl
#3. Bümpliz-Oberbottigen

# Quartiere
#1. Schosshalde
#2. Bümpliz
#3. Weissenbühl
#4. Innere Stadt
df_auslaender.sort_values(by='uk', ascending=False).head(10)

quartier_stadtteil  total_2018  deutschland  mazedonien  spanien  frankreich  italien  eritrea  kosovo  kroatien  oesterreich  polen  portugal  serbien  tuerkei   uk  somalia  usa  china  indien  srilanka  syrien
38          stadtbern_total       34375         6345        1238     1970         863     4200      841    1230       471          649    463      1551      737     1182  420      474  371    653     391       812     474
26  Kirchenfeld-Schosshalde        5501         1288          44      240         175      483       63      76        42          119     87       169       89      101  124       41  100    149      96        62      56
19    Mattenhof-Weissenbühl        7189         1800          73      374         252      812      162     119        82          153    121       215      117      181   98       77   88    168      99       201     101
37     Bümpliz-Oberbottigen       12189          934        1018      742          98     1733      398     802       224          128    115       876      382      705   61      309   42    206     110       334     211
32     Breitenrain-Lorraine        4837         1038          58      373         150      614      110     182        72          114     82       173       85      120   56       30   62     74      30       126      47
12       Länggasse-Felsenau        3769         1032          45      212         140      441      106      49        47          113     42        98       58       58   54       15   41     51      55        76      55
24              Schosshalde        1842          446          24       72          61      191       13      36        16           41     20        57       58       38   49        7   29     44      19        20      24
33                  Bümpliz        5612          445         443      358          56      877      203     350       120           68     55       370      155      269   37      111   24     97      50       182      78
17              Weissenbühl        1483          404           7       77          46      160       39      20         7           36     26        29       43       41   36        4   14     32      14        20      13
5              Innere Stadt         890          253           0       29          48      117        2       2         4           22     16        20        6       17   27        2   38      5       1        13       4

In [83]:
(100/420)*124 # In Kirchenfeld-Schosshalde

29.523809523809522

In [84]:
(100/420)*98 # In Mattenhof-Weissenbühl

23.333333333333332

In [85]:
(100/420)*61 # In Bümpliz-Oberbottigen

14.523809523809524

In [86]:
(100/420)*49 # In Schosshalde

11.666666666666666

In [87]:
(100/420)*37 # In Bümpliz

8.809523809523808

In [88]:
(100/420)*36 # In Weissenbühl

8.571428571428571

### 5. Frankreich 

In [89]:
# Beliebteste Stadtteile für Franzosen:
#1. Mattenhof-Weissenbühl
#2. Kirchenfeld-Schosshalde
#3. Breitenrain-Lorraine
#4. Länggasse-Felsenau

# Quartiere
#1. Monbijou
#2. Schosshalde
#3. Bümpliz
#4. Breitenrain
df_auslaender.sort_values(by='frankreich', ascending=False).head(10)

quartier_stadtteil  total_2018  deutschland  mazedonien  spanien  frankreich  italien  eritrea  kosovo  kroatien  oesterreich  polen  portugal  serbien  tuerkei   uk  somalia  usa  china  indien  srilanka  syrien
38          stadtbern_total       34375         6345        1238     1970         863     4200      841    1230       471          649    463      1551      737     1182  420      474  371    653     391       812     474
19    Mattenhof-Weissenbühl        7189         1800          73      374         252      812      162     119        82          153    121       215      117      181   98       77   88    168      99       201     101
26  Kirchenfeld-Schosshalde        5501         1288          44      240         175      483       63      76        42          119     87       169       89      101  124       41  100    149      96        62      56
32     Breitenrain-Lorraine        4837         1038          58      373         150      614      110     182        72          114     82       173       85      120   56       30   62     74      30       126      47
12       Länggasse-Felsenau        3769         1032          45      212         140      441      106      49        47          113     42        98       58       58   54       15   41     51      55        76      55
37     Bümpliz-Oberbottigen       12189          934        1018      742          98     1733      398     802       224          128    115       876      382      705   61      309   42    206     110       334     211
16                 Monbijou         752          216           5       32          65       66        5      11         9           15     13        11       10        8   14        2   21      7       1        13       7
24              Schosshalde        1842          446          24       72          61      191       13      36        16           41     20        57       58       38   49        7   29     44      19        20      24
33                  Bümpliz        5612          445         443      358          56      877      203     350       120           68     55       370      155      269   37      111   24     97      50       182      78
30              Breitenrain        1384          235          15      101          53      179       45      91        22           28     17        54       16       43   13        5   10     19       8        25       5

In [90]:
#1. Mattenhof-Weissenbühl
(100/863)*252

29.20046349942063

In [91]:
#2. Kirchenfeld-Schosshalde
(100/863)*175

20.278099652375435

In [92]:
#3. Breitenrain-Lorraine
(100/863)*150

17.38122827346466

In [93]:
#4. Länggasse-Felsenau
(100/863)*140

16.22247972190035

In [94]:
# Quartiere
#1. Monbijou
(100/863)*65

7.531865585168019

In [95]:
#2. Schosshalde
(100/863)*61

7.068366164542295

In [97]:
#3. Bümpliz
(100/863)*56

6.48899188876014

### 6. USA 

In [100]:
# Beliebteste Stadtteile für US-Amerikaner:
#1. Kirchenfeld-Schosshalde
#2. Mattenhof-Weissenbühl
#3. Breitenrain-Lorraine

# Quartiere
#1. Innere Stadt
#2. Schosshalde
#3. Spitalacker
df_auslaender.sort_values(by='usa', ascending=False).head(10)

quartier_stadtteil  total_2018  deutschland  mazedonien  spanien  frankreich  italien  eritrea  kosovo  kroatien  oesterreich  polen  portugal  serbien  tuerkei   uk  somalia  usa  china  indien  srilanka  syrien
38          stadtbern_total       34375         6345        1238     1970         863     4200      841    1230       471          649    463      1551      737     1182  420      474  371    653     391       812     474
26  Kirchenfeld-Schosshalde        5501         1288          44      240         175      483       63      76        42          119     87       169       89      101  124       41  100    149      96        62      56
19    Mattenhof-Weissenbühl        7189         1800          73      374         252      812      162     119        82          153    121       215      117      181   98       77   88    168      99       201     101
32     Breitenrain-Lorraine        4837         1038          58      373         150      614      110     182        72          114     82       173       85      120   56       30   62     74      30       126      47
37     Bümpliz-Oberbottigen       12189          934        1018      742          98     1733      398     802       224          128    115       876      382      705   61      309   42    206     110       334     211
12       Länggasse-Felsenau        3769         1032          45      212         140      441      106      49        47          113     42        98       58       58   54       15   41     51      55        76      55
5              Innere Stadt         890          253           0       29          48      117        2       2         4           22     16        20        6       17   27        2   38      5       1        13       4
24              Schosshalde        1842          446          24       72          61      191       13      36        16           41     20        57       58       38   49        7   29     44      19        20      24
28              Spitalacker        1155          347           1       80          39      122       20      13         9           36     23        23        9       20   24        1   24     33       9        24      18
33                  Bümpliz        5612          445         443      358          56      877      203     350       120           68     55       370      155      269   37      111   24     97      50       182      78

In [105]:
#1. Kirchenfeld-Schosshalde
(100/371)*100

26.954177897574123

In [102]:
#2. Mattenhof-Weissenbühl
(100/371)*88

23.719676549865227

In [103]:
#3. Breitenrain-Lorraine
(100/371)*62

16.711590296495956

In [106]:
# Quartiere
#1. Innere Stadt
(100/371)*38

10.242587601078167

In [107]:
#2. Schosshalde
(100/371)*29

7.816711590296495

In [108]:
#3. Spitalacker
(100/371)*24

6.469002695417789

In [ ]:
100+100